src/sentry/models/projectownership.py

```diff
@@ -96,37 +96,45 @@ def get_owners(cls, project_id, data):
         return ordered_actors, rules
 
     @classmethod
-    def get_autoassign_owner(cls, project_id, data):
+    def get_autoassign_owners(cls, project_id, data, limit=2):
         """
         Get the auto-assign owner for a project if there are any.
 
         Will return None if there are no owners, or a list of owners.
```

Comment:
does this comment need to be changed to reflect new return type?

Reply:
Yeah, good catch!